src/app/[locale]/(components)/serverSubmitHandlers.tsx

```diff
@@ -16,31 +16,54 @@ export const loginSubmit = async (data: FormData) => {
     // @ts-ignore
 
     cookies().set({
-      name: "session",
+      name: 'session',
       value: resp.tokenCreate?.token,
       httpOnly: true,
-      path: "/",
+      path: '/',
       expires: new Date(Date.now() + twentyFourHoursInSeconds * 1000),
     });
   }
 
   return resp;
 };
+
 export const registerSubmit = async (data: FormData) => {
-  const firstName = data.get("firstName") as string;
-  const lastName = data.get("lastName") as string;
-  const email = data.get("email") as string;
-  const password = data.get("password") as string;
+  const firstName = data.get('firstName') as string;
+  const lastName = data.get('lastName') as string;
+  const email = data.get('email') as string;
+  const password = data.get('password') as string;
 
   const resp = await gqlClient.accountRegister({
     input: {
       email,
       password,
       firstName,
       lastName,
-      channel: "default-channel",
+      channel: 'default-channel',
     },
   });
 
   return resp;
 };
+
+export const requestPasswordReset = async (data: FormData) => {
+  const email = data.get('email') as string;
+  const redirectUrl = 'http://localhost:3000/reset-password';
```

Comment:
can this be set to the current domain somehow? In production, this would need to match the host/origin its deployed to instead of `localhost:3000`